Libraries/Azure.psm1

```diff
@@ -426,13 +426,25 @@ Function Invoke-AllResourceGroupDeployments($SetupTypeData, $CurrentTestData, $R
 			$DiskType = "Unmanaged"
 		}
 
+		if (!$global:password -and !$global:sshPrivateKey) {
```

Comment:
Just curious how this is happening.

Reply:
some one may not know. currently lack of both won't throw exception.